docs/.vuepress/theme/components/Logo.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+      <svg viewBox="0 0 1300 197.33" xmlns="http://www.w3.org/2000/svg" id="handsontable-logo">
+        <defs>
+        <style>.cls-1{fill:#1a42e8;}</style>
+        </defs>
+        <title>Handsontable_logo_color</title>
+        <path transform="translate(-50 -101.33)" d="m50 108.22h37.31v49.447h42.56v-49.447h37.125v129.49h-37.125v-51.068h-42.56v51.068h-37.31z"/>
+        <path transform="translate(-50 -101.33)" d="m182.72 211.45c0-22.271 18.838-30.969 50.348-30.969h9.0588v-2.172c0-9.0482-2.172-15.034-12.497-15.034-8.8734 0-12.137 5.2551-12.857 11.231h-30.429c1.4515-22.822 19.378-34.042 45.278-34.042 26.08 0 42.74 10.68 42.74 35.854v61.398h-31.51v-11.051c-4.5294 6.8868-12.862 13.223-28.072 13.223-17.026 0-32.061-8.3277-32.061-28.437zm59.406-6.8762v-5.7955h-8.5131c-13.037 0-18.833 2.5322-18.833 10.32 0 5.4353 3.6182 8.8786 11.225 8.8786 9.2389 3e-5 16.12-5.2551 16.12-13.403z"/>
```

Comment:
Can we somehow suppress the linter error for this file? Disable maximum line rule for Logo.vue or break lines